libraries/hotstuff/chain_pacemaker.cpp

```diff
@@ -112,9 +112,31 @@ namespace eosio { namespace hotstuff {
       return _chain->is_builtin_activated( builtin_protocol_feature_t::instant_finality );
    }
 
-   void chain_pacemaker::get_state( finalizer_state& fs ) const {
-      if (enabled())
-         _qc_chain.get_state( fs ); // get_state() takes scare of finer-grained synchronization internally
+   void chain_pacemaker::get_state(finalizer_state& fs, bool force) const {
```

Comment:
I don't see that `force` is ever `true`. I think it should be removed if not used.